src/lib/config.ts

```diff
@@ -66,6 +66,27 @@ class default_config {
     priority = "0"
 
     // Note to developers: When creating new <modifier-letter> maps, make sure to make the modifier uppercase (e.g. <C-a> instead of <c-a>) otherwise some commands might not be able to find them (e.g. `bind <c-a>`)
+    
+    /**
+     * cmdmaps contains all of the bindings for the command line.
+     */
+    exmaps = {
+        "<Enter>": "ex_accept_line",
+        "<C-j>": "ex_accept_line",
+        "<C-m>": "ex_accept_line",
+        "<Escape>": "ex_hide_and_clear",
+        "<ArrowUp>": "ex_previous_history",
+        "<ArrowDown>": "ex_next_history",
+        "<C-a>": "ex_beginning_of_line",
```

Comment:
Can't these just reuse the im_* equivalents?

Reply:
im_* and ex_* functions use the same functions to compute modifications (e.g. both `ex_beginning_of_line` and `im_beginning_of_line` use `editor.ts:beginning_of_line`)  but they still need to act on different inputs, the command-line input isn't available to the content script and content script inputs aren't available to the command line.